src/css/controls/flags/ads.less

```diff
@@ -96,13 +56,8 @@
         }
     }
 
-    .jw-group > .jw-text-alt {
-        display: inline-block;
-        margin: 0;
-        line-height: @mobile-touch-target;
-    }
-
-    .jw-text-duration {
+    .jw-text-duration,
```

Comment:
You can group these with the group of classes above ending with `.jw-autostart-mute`